### ISSUE TITLE
Cannot log in to ems.element.io hosted servers

### ISSUE THREAD

**Issue by MatejLach** [mirukana/mirage#184]
First of all, thanks a lot for your work on Mirage. 

I ran into a curious issue where when trying to login to a server hosted on https://ems.element.io, Mirage simply fails with an invalid access token error and logs out. 

```
Traceback (most recent call last):
  File "qrc:/src/backend/qml_bridge.py", line 83, in on_done
    result = future.result()
  File "/usr/lib/python3.9/concurrent/futures/_base.py", line 433, in result
    return self.__get_result()
  File "/usr/lib/python3.9/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "qrc:/src/backend/matrix_client.py", line 1658, in devices_info
    for device in (await self.devices()).devices
  File "/usr/lib/python3.9/site-packages/nio/client/async_client.py", line 1363, in devices
    return await self._send(DevicesResponse, method, path)
  File "qrc:/src/backend/matrix_client.py", line 255, in _send
    raise MatrixError.from_nio(response)
backend.errors.MatrixInvalidAccessToken
```

I've also tested logging in with Element & nheko and they both log in successfully, so I ruled out a server configuration issue. 

Steps to reproduce:

1. Create a server on  ems.element.io (trial available or am happy to provide an account for testing).

2. Try to log in to the server with Mirage.

**Comment by mirukana** [comment]
Which version of Mirage are you running? This might be due to a bug that was fixed by eaf96b1d62e9fa3f633c15ff2459853708f5b521, if you can try the dev branch. In other cases, I would like an account for testing, since starting the trial requires personal details.

**Comment by MatejLach** [comment]
Ok, yeah, that commit fixes it indeed.. 

I am testing this on the PinePhone, (Mobian specifically), where the easiest way to install it is via apt[1], which is however tied to tagged releases. 

Thanks for the pointer and all your efforts on Mirage.

1 - https://packages.debian.org/unstable/net/matrix-mirage
